lms/views/api/error.py

```diff
@@ -39,6 +39,14 @@ def canvas_api_error(context, request):
     return {"error_message": context.explanation, "details": context.details}
 
 
+@exception_view_config(context=LTIOutcomesAPIError, renderer="json")
+def lti_outcomes_api_error(context, request):
+    request.response.status_int = 400
+    # Send the frontend an error message and details to show to the user for
+    # debugging.
+    return {"error_message": context.explanation, "details": context.details}
```

Comment:
I think this exception view is exactly the same as the `canvas_api_error()` one above it? It might be possible to rename `canvas_api_error()` to something more generic and have it handle both types of exception?

```python
@exception_view_config(context=CanvasAPIError, renderer="json")
@exception_view_config(context=LTIOutcomesAPIError, renderer="json")
def proxy_api_error(context, request):
    request.response.status_int = 400
    # Send the frontend an error message and details to show to the user for
    # debugging.
    return {"error_message": context.explanation, "details": context.details}
```